### DISCUSSION TITLE
Cannot install Deta CLI

### DISCUSSION THREAD

**Discussion by remosingh** [deta/cloud-docs#669] [Help]
this is the error I am getting even after successfully installing Deta

<img width="621" alt="Screenshot 2022-12-20 at 23 50 58" src="https://user-images.githubusercontent.com/36945245/208849971-20e804d1-a480-4151-9be5-7876d08a110f.png">

**Comment by sudo-subg** [comment]
Seems like the downloaded binary which contains the CLI code failed to export the `deta` command to your path. You have to add that by yourself manually.

**Reply by remosingh** [reply]
Hi mik,

Thanks for the response. I am sorry but what do you mean by adding it manually? What do I add where?

Remo

**Reply by sudo-subg** [reply]
Add `export PATH="~/.deta/bin:$PATH"` in your .bashrc file which you can find in your home directory.

**Reply by remosingh** [reply]
Done. Still nothing.....

Do I have to run it somewhere before I use it in VS Code?

<img width="526" alt="Screenshot 2022-12-24 at 14 23 23" src="https://user-images.githubusercontent.com/36945245/209452246-52ef5118-5434-4ad0-8c2b-17ad93611cb6.png">
